antichat/antichat.lua

```diff
@@ -0,0 +1,85 @@
+modname="Antichat"
+version="1.0"
+
+function et_InitGame()
+        et.RegisterModname(modname.." "..version)
+end
+-- Antichat by Ryven
+-- Allow only wolfadmin commands to be passed
+
+local blackList = { "say", "say_team", "say_buddy", "say_teamnl", "m", "pm" }
+local whitelist = {
```

Comment:
What's the point in hardcoding wolfadmin stuff, like why being responsible for maintaining the list? Instead you could just suggest people to put wofladmin first to load in the lua modules list, then it will be able to parse its own stuff first, then passing control further here.

Reply:
Also, there is `blackList` with uppercase `L` and also `whitelist` with lowercase `l`. This should be standarized to lowercase probably ;)